src/components/Home/HomeSlider.jsx

```diff
@@ -1,181 +1,177 @@
+                  </div>
+                </div>
+                {/* story 2 */}
+                <div className="flex flex-col flex-shrink-0 w-auto overflow-hidden rounded-lg hover-scale shadow-button">
+                  <div className="flex flex-col flex-1 justify-between py-4 px-10 bg-white">
+                    <div className="flex-1">
+                      <a href="/#" className="block text-center">
+                        <span className="text-xl 2xl:text-3xl font-normal leading-tight leading-7 text-black">
+                          Mueble
+                        </span>
+                      </a>
+                    </div>
+                  </div>
+                </div>
+                {/* story 3 */}
+                <div className="flex flex-col flex-shrink-0 w-auto overflow-hidden rounded-lg hover-scale shadow-button">
+                  <div className="flex flex-col flex-1 justify-between py-4 px-10 bg-white">
+                    <div className="flex-1">
+                      <a href="/#" className="block text-center">
+                        <span className="text-xl 2xl:text-3xl font-normal leading-tight leading-7 text-black">
+                          Mueble
+                        </span>
+                      </a>
+                    </div>
+                  </div>
+                </div>
+                {/* story 4 */}
+                <div className="flex flex-col flex-shrink-0 w-auto overflow-hidden rounded-lg hover-scale shadow-button">
+                  <div className="flex flex-col flex-1 justify-between py-4 px-10 bg-white">
+                    <div className="flex-1">
+                      <a href="/#" className="block text-center">
+                        <span className="text-xl 2xl:text-3xl font-normal leading-tight leading-7 text-black">
+                          Mueble
+                        </span>
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              {/* End slide page 2 */}
+            </div>
+            {/* End slide page 2 */}
+            <div className="relative flex flex-col items-end justify-center w-auto">
+              <div className="relative flex space-x-2">
+                <div className="flex items-center justify-center w-10 h-10 text-black bg-white rounded-full cursor-pointer">
+                  <svg
+                    className="w-5 h-5"
+                    fill="none"
+                    stroke="currentColor"
+                    viewBox="0 0 24 24"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      strokeWidth={2}
+                      d="M9 5l7 7-7 7"
+                    />
+                  </svg>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
```

Comment:
This slider need to be changed to the new categories list, if is much effort I can open a new issue for this

Reply:
Please open a new one